client/src/pages/GeneralInfo/GeneralInfo.tsx

```diff
@@ -2,6 +2,8 @@ import styles from './GeneralInfo.module.scss';
 import { useState } from 'react';
 import { Button, Box, Stack, Typography } from '@mui/material';
```

Comment:
I don't see Button or Typography used. It's probably best to remove them? 
Also instead of using the mui Button please use the custom button for consistency.

Reply:
Thank you Dafne, I believe this code was changed. I just merged with main.